### PR TITLE
feat(connector): add bonita-connector-model module

### PR DESCRIPTION
**Pull request by akantcheff**
Add module `bonita-connector-model` that defines the Bonita Connector Model.

Is part of [DEV-453](https://bonitasoft.atlassian.net/browse/DEV-453)

[DEV-453]: https://bonitasoft.atlassian.net/browse/DEV-453?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: connector-model/src/main/resources/user-filter-definition-descriptor.xsd [hunk 1]
```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (rbioteau) [inline connector-model/src/main/resources/user-filter-definition-descriptor.xsd:1]: WE don't want this one. Like for implementation, only connector definition should be used, even for filters

**Review by rbioteau — CHANGES_REQUESTED** [review]

File: connector-model/src/main/resources/connector-definition-descriptor.xsd [hunk 2]
```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<xsd:schema xmlns="http://www.bonitasoft.org/ns/connector/definition/6.0" xmlns:xsd="http://www.w3.org/2001/XMLSchema" targetNamespace="http://www.bonitasoft.org/ns/connector/definition/6.0">
```
Comment (rbioteau) [inline connector-model/src/main/resources/connector-definition-descriptor.xsd:2]: this is not the proper version. Should be `http://www.bonitasoft.org/ns/connector/definition/6.1`

Comment (akantcheff) [reply connector-model/src/main/resources/connector-definition-descriptor.xsd:2]: We also need to update it here https://github.com/bonitasoft/bonita-process-model/blob/develop/bundles/org.bonitasoft.bonita2bar/src/org/bonitasoft/bonita2bar/process/ProcessRegistryImpl.java#L67

Comment (rbioteau) [reply connector-model/src/main/resources/connector-definition-descriptor.xsd]: This is like this on purpose for now. The migration in proc file is not done properly and we still have references to the old version.
https://bonitasoft.atlassian.net/browse/DEV-455 should tackle this.

**Review by rbioteau — APPROVED** [review]

**Pull request merged**